dev-client/package.json

```diff
@@ -11,8 +11,11 @@
   },
   "dependencies": {
     "@rnmapbox/maps": "^10.0.0",
+    "native-base": "^3.4.28",
     "react": "18.2.0",
-    "react-native": "0.71.6"
+    "react-native": "0.71.6",
+    "react-native-safe-area-context": "^3.3.2",
+    "react-native-svg": "^12.1.1"
```

Comment:
As per suggested in the [nativebase docs](https://docs.nativebase.io/install-rn#h3-install-dependencies)